Mixpanel/Mixpanel.m

```diff
@@ -28,7 +29,7 @@
 #error The Mixpanel library must be compiled with ARC enabled
 #endif
 
-#define VERSION @"3.3.9"
+#define VERSION @"3.4.0"
```

Comment:
version bump is not needed in the PR, this will be added after the PR being merged.